packages/libmikmod/libmikmod.spec.origfedora

```diff
@@ -0,0 +1,249 @@
+Summary:        A MOD music file player library
```

Comment:
General comment is that this `.spec.origfedora` is in the wrong place - it is currently packages/libmikmod/libmikmod.spec.origfedora and should be packages/libmikmod/SPECS/libmikmod.spec.origfedora

Reply:
`.spec.origfedora` added - thanks - could you remove the one in the wrong place too pls. thanks.